src/authentication/BasicCredentialsExtractor.ts

```diff
@@ -5,7 +5,7 @@ import { CredentialsExtractor } from './CredentialsExtractor';
 /**
  * Credentials extractor which simply interprets the contents of the Authorization header as a webID.
  */
-export class SimpleCredentialsExtractor extends CredentialsExtractor {
+export class BasicCredentialsExtractor extends CredentialsExtractor {
```

Comment:
I'd name it based on what it does. Perhaps `UnsecureWebIdExtractor` or `PlainTextWebIdExtractor`?

Reply:
PS the method below can just be `return { webID: input.headers.authorization };`, I think.

Reply:
Yea true, I think this might be because this was written before we had strict settings